Automattic/wp-calypso
# Command Palette: Add Visit Site Command

Related to p1706557832374639/1706325532.678219-slack-C0347E545HR.

## Proposed Changes

Adds a command that lets the user visit the current site, or choose one to visit, quickly.

| From a global page | Within a specific site |
| -------------------|-----------------------|
| <img width="494" alt="image" src="https://github.com/Automattic/wp-calypso/assets/26530524/b0fc6178-9b2e-427d-8b1d-26970f91543e"> | <img width="460" alt="image" src="https://github.com/Automattic/wp-calypso/assets/26530524/e6d10658-01ea-42ef-a235-819bcaf8c341"> |

## Testing Instructions

From `/sites`, open the command palette and type "Visit site". Input any site and press Enter. You should see the front end of the chosen site.

From `/home/%s`, open the command palette and type "Visit site". Press Enter, and you should see the current site's front end.
